src/app/pages/courses/course.component.html

```diff
@@ -0,0 +1,84 @@
+<div class="course">
```

Comment:
Z tego co wiem, nie trzeba było w zadaniu tworzyć strony z formularzem dodawania nowego kursu, ale jak coś to nazwa pliku powinna to mówić, zamiast `course.component` powinno być np. `addCourse.component`. 
Dodatkowo jest mały mish mash bo w `pages` masz folder `courses` (liczba mnoga) co sugerowałoby np listę kursów, a w środku masz komponent dodawania o nazwie `course.component`, którego selektor to `'find-course'`. List kursów jest natomiast w `home.component`